### ISSUE TITLE
Congenital seminal vesicle cyst

### ISSUE THREAD

**Issue by jagcgc** [obophenotype/human-phenotype-ontology#10270]
> I can see this term was previously requested [here](https://github.com/obophenotype/human-phenotype-ontology/issues/8907), but I did not see that it was created, so I would like to re-request.

**Preferred term label:**
Congenital seminal vesicle cyst

**Synonyms**
- Cysts of the seminal vesicles
- Cysts involving the seminal vesicles
- Seminal vesicle cysts
- Seminal vesicular cysts

**Definition (free text, please give PubMed ID)**
A cystic structure caused by distention of the seminal vesicle due to atresia of the ejaculatory duct. Caused by altered development of the mesonephric duct (AKA Wolffian duct). Two-thirds are associated with abnormal development of other ipsilateral mesonephric derivatives, such as vas deferens, kidney, and ureter. Sometimes considered the male equivalent of Mayer-Rokitansky-Kuster-Hauser syndrome. ([PMID: 31024871](https://pubmed.ncbi.nlm.nih.gov/31024871))

**Parent term (use hpo.jax.org/app)**
Abnormal male reproductive system physiology [HP:0012874](https://hpo.jax.org/app/browse/term/HP:0012874)

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
Zinner syndrome (no Orphanet or OMIM entry) ([Reference: PMID: 34268266](https://pubmed.ncbi.nlm.nih.gov/34268266))
Autosomal dominant polycystic kidney disease ([ORPHA:730](https://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=EN&Expert=730)) ([Reference: PMID: 9829485](https://doi.org/10.1093/ndt/13.11.2825))
Renal agenesis ([ORPHA:411709](https://www.orpha.net/consor/cgi-bin/OC_Exp.php?Lng=EN&Expert=411709))

**Your nano-attribution (ORCID)**
0000-0001-8481-3559

**Comment by pnrobinson** [comment]
@jagcgc adding without the "congenital" modifier since this is an inference rather than a direct phenotypic observation.
thanks!

Adding new term: 
- ID: HP:6000191
- label: Seminal vesicle cyst
- parents: [Abnormal male internal genitalia morphology (HP:0000022)](https://hpo.jax.org/app/browse/term/HP:0000022)
- pmid: PMID:31024871|PMID:9829485
- def: A cyst (sac-like mass that may contain serous liquid or semisolid material) caused by distention of the seminal vesicle due to atresia of the ejaculatory duct.
- comment: Congenital seminal vesical cysts are caused by altered development of the mesonephric (Wolffian) duct. They are often associated with abnormal development of other ipsilateral mesonephric derivatives, such as vas deferens, kidney, and ureter.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10270
- synonyms:
